runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordState.java

```diff
@@ -126,7 +126,7 @@ void triggerEvent(final DiscordGameEventType eventType)
 		{
 			// If we aren't showing the elapsed time within Discord then
 			// We null out the event start property
```

Comment:
this comment is not relevant anymore I think?

Reply:
Removed